package.json

```diff
@@ -870,7 +870,7 @@
         "shell-env": "^0.3.0",
         "shelljs": "0.7.7",
         "styled-components": "^2.3.0",
-        "typescript": "2.7.1",
+        "typescript": "^2.8.1",
```

Comment:
Just curious - was there a part of this change that required moving to TypeScript 2.8?

Reply:
That was the result of a very long story involving trying to neaten the typing of the epics since there are a lot of quirks that result in type casting, I left it in as I thought it would spare someone the trouble of having to update ts later on (has proven stable with no issues)